pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
```

Comment:
This file should be part of PR #34 
please remove it from this PR

Reply:
done